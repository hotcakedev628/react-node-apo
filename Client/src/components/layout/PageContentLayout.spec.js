import renderer from 'react-test-renderer';
import { PageContentLayout } from './PageContentLayout';
import React from 'react'


describe('PageContentLayout', () => {
    it('should match snapshot', () => {
      const tree = renderer.create(<PageContentLayout  isRendering={true} unAvailabilityText="not available"> <li>Hey</li></PageContentLayout>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  