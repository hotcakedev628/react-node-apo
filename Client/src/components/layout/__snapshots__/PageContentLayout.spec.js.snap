// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContentLayout should match snapshot 1`] = `
Array [
  " ",
  <li>
    Hey
</li>,
]
`;
